aws-sdk-s3-transfer-manager/src/http.rs

```diff
@@ -2,14 +2,5 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Should we just remove this file for now?

Reply:
It still contains the needed module declaration (line 6) `pub(crate) mod header` for the HTTP header utils